jnigen/lib/src/config/config_types.dart

```diff
@@ -619,6 +652,19 @@ class Config {
   }
 }
 
+enum Experiment { interfaceImplementation }
```

Comment:
Can we make this a class with two fields `configString` and `description`?

For reference (we probably don't need something as big as these two).

Dart experiments:

```dart
  nativeAssets(
      name: 'native-assets',
      isEnabledByDefault: false,
      isExpired: false,
      experimentEnabledVersion: const Version(3, 1),
      experimentReleasedVersion: const Version(3, 1)),
```

Flutter experiments:

```dart
const Feature nativeAssets = Feature(
  name: 'native assets compilation and bundling',
  configSetting: 'enable-native-assets',
  environmentOverride: 'FLUTTER_NATIVE_ASSETS',
  master: FeatureChannelSetting(
    available: true,
  ),
);
```